@corbinrobb can you fix these issues to allow CI to pass
```
/home/runner/work/superset/superset/superset-frontend/src/SqlLab/components/TableElement.jsx
   35:1   error  `src/components/Icons` import should occur before import of `../../components/CopyToClipboard`         import/order
   70:9   error  'popSelectStar' is assigned a value but never used                                                     no-unused-vars
  192:30  error  Unexpected block statement surrounding arrow body; move the returned value immediately after the `=>`  arrow-body-style
  264:36  error  Unexpected block statement surrounding arrow body; move the returned value immediately after the `=>`  arrow-body-style
  280:20  error  Curly braces are unnecessary here                                                                      react/jsx-curly-brace-presence
  292:7   error  Value must be omitted for boolean attributes                                                           react/jsx-boolean-value
```

--- a/superset-frontend/spec/javascripts/sqllab/TableElement_spec.jsx
+++ b/superset-frontend/spec/javascripts/sqllab/TableElement_spec.jsx
@@ -62,7 +62,7 @@ describe('TableElement', () => {
     expect(wrapper.find(ColumnElement)).toHaveLength(14);
   });
   it('mounts', () => {
-    mount(
+    const wrapper = mount(
       <Provider store={store}>
         <TableElement {...mockedProps} />
       </Provider>,
@@ -73,6 +73,8 @@ describe('TableElement', () => {
         },
       },
     );
+
+    expect(wrapper.find(TableElement)).toHaveLength(1);
   });
   it('fades table', async () => {
     const wrapper = mount(

--- a/superset-frontend/src/SqlLab/components/TableElement.jsx
+++ b/superset-frontend/src/SqlLab/components/TableElement.jsx
@@ -21,18 +21,17 @@ import PropTypes from 'prop-types';
 import Collapse from 'src/components/Collapse';
 import Card from 'src/components/Card';
 import ButtonGroup from 'src/components/ButtonGroup';
-import shortid from 'shortid';
 import { t, styled } from '@superset-ui/core';
 import { debounce } from 'lodash';
 
 import { Tooltip } from 'src/components/Tooltip';
+import Icons from 'src/components/Icons';
 import CopyToClipboard from '../../components/CopyToClipboard';
 import { IconTooltip } from '../../components/IconTooltip';
 import ColumnElement from './ColumnElement';
 import ShowSQL from './ShowSQL';
 import ModalTrigger from '../../components/ModalTrigger';
 import Loading from '../../components/Loading';
-import Icons from 'src/components/Icons';
 
 const propTypes = {
   table: PropTypes.object,
@@ -65,17 +64,6 @@ const TableElement = props => {
 
   const setHover = hovered => {
     debounce(() => setHovered(hovered), 100)();
-  };
-
-  const popSelectStar = () => {
-    const qe = {
-      id: shortid.generate(),
-      title: table.name,
-      dbId: table.dbId,
-      autorun: true,
-      sql: table.selectStar,
-    };
-    actions.addQueryEditor(qe);
   };
 
   const removeTable = () => {
@@ -189,41 +177,39 @@ const TableElement = props => {
     );
   };
 
-  const renderHeader = () => {
-    return (
-      <div
-        className="clearfix header-container"
-        onMouseEnter={() => setHover(true)}
-        onMouseLeave={() => setHover(false)}
+  const renderHeader = () => (
+    <div
+      className="clearfix header-container"
+      onMouseEnter={() => setHover(true)}
+      onMouseLeave={() => setHover(false)}
+    >
+      <Tooltip
+        id="copy-to-clipboard-tooltip"
+        placement="topLeft"
+        style={{ cursor: 'pointer' }}
+        title={table.name}
+        trigger={['hover']}
       >
-        <Tooltip
-          id="copy-to-clipboard-tooltip"
-          placement="topLeft"
-          style={{ cursor: 'pointer' }}
-          title={table.name}
-          trigger={['hover']}
-        >
-          <StyledSpan data-test="collapse" className="table-name">
-            <strong>{table.name}</strong>
-          </StyledSpan>
-        </Tooltip>
+        <StyledSpan data-test="collapse" className="table-name">
+          <strong>{table.name}</strong>
+        </StyledSpan>
+      </Tooltip>
 
-        <div className="pull-right header-right-side">
-          {table.isMetadataLoading || table.isExtraMetadataLoading ? (
-            <Loading position="inline" />
-          ) : (
-            <Fade
-              data-test="fade"
-              hovered={hovered}
-              onClick={e => e.stopPropagation()}
-            >
-              {renderControls()}
-            </Fade>
-          )}
-        </div>
+      <div className="pull-right header-right-side">
+        {table.isMetadataLoading || table.isExtraMetadataLoading ? (
+          <Loading position="inline" />
+        ) : (
+          <Fade
+            data-test="fade"
+            hovered={hovered}
+            onClick={e => e.stopPropagation()}
+          >
+            {renderControls()}
+          </Fade>
+        )}
       </div>
-    );
-  };
+    </div>
+  );
 
   const renderBody = () => {
     let cols;
@@ -261,35 +247,33 @@ const TableElement = props => {
     return metadata;
   };
 
-  const collapseExpandIcon = () => {
-    return (
-      <IconTooltip
-        style={{
-          position: 'fixed',
-          right: '16px',
-          left: 'auto',
-          fontSize: '12px',
-          transform: 'rotate(90deg)',
-          display: 'flex',
-          alignItems: 'center',
-        }}
-        aria-label="Collapse"
-        tooltip={t(`${!isActive ? 'Expand' : 'Collapse'} table preview`)}
-      >
-        <Icons.RightOutlined
-          iconSize={'s'}
-          style={isActive ? { transform: 'rotateY(180deg)' } : null}
-        />
-      </IconTooltip>
-    );
-  };
+  const collapseExpandIcon = () => (
+    <IconTooltip
+      style={{
+        position: 'fixed',
+        right: '16px',
+        left: 'auto',
+        fontSize: '12px',
+        transform: 'rotate(90deg)',
+        display: 'flex',
+        alignItems: 'center',
+      }}
+      aria-label="Collapse"
+      tooltip={t(`${!isActive ? 'Expand' : 'Collapse'} table preview`)}
+    >
+      <Icons.RightOutlined
+        iconSize="s"
+        style={isActive ? { transform: 'rotateY(180deg)' } : null}
+      />
+    </IconTooltip>
+  );
 
   return (
     <Collapse.Panel
       {...props}
       header={renderHeader()}
       className="TableElement"
-      forceRender={true}
+      forceRender
       expandIcon={collapseExpandIcon}
     >
       {renderBody()}

--- a/superset-frontend/src/SqlLab/components/TableElement.jsx
+++ b/superset-frontend/src/SqlLab/components/TableElement.jsx
@@ -112,7 +112,7 @@ const TableElement = props => {
 
   const renderControls = () => {
     let keyLink;
-    if (table?.indexes.length > 0) {
+    if (table?.indexes?.length) {
       keyLink = (
         <ModalTrigger
           modalTitle={
@@ -137,7 +137,7 @@ const TableElement = props => {
         {keyLink}
         <IconTooltip
           className={
-            `fa fa-sort-${!sortColumns ? 'alpha' : 'numeric'}-asc ` +
+            `fa fa-sort-${sortColumns ? 'numeric' : 'alpha'}-asc ` +
             'pull-left sort-cols m-l-2 pointer'
           }
           onClick={toggleSortColumns}
@@ -259,7 +259,7 @@ const TableElement = props => {
         alignItems: 'center',
       }}
       aria-label="Collapse"
-      tooltip={t(`${!isActive ? 'Expand' : 'Collapse'} table preview`)}
+      tooltip={t(`${isActive ? 'Collapse' : 'Expand'} table preview`)}
     >
       <Icons.RightOutlined
         iconSize="s"

